app/presenters/chf_index_presenter.rb

```diff
@@ -0,0 +1,26 @@
+class ChfIndexPresenter < Blacklight::IndexPresenter
+  class_attribute :description_max_chars
+  self.description_max_chars = 10
```

Comment:
oops, that was an accident for testing, I meant it to be 500. I can be whatever we like, and easily changed later. Will fix it to 500 for now.